README.rst

```diff
@@ -10,179 +10,14 @@ vulkan-renderer
 
 ----
 
-A new octree-based game+engine using `Vulkan API <https://www.khronos.org/vulkan/>`__ and `C++17 <https://stackoverflow.com/questions/38060436/what-are-the-new-features-in-c17>`__.
-
-Our current octree demo ``v0.1-alpha.2`` can be downloaded `here <https://github.com/inexorgame/vulkan-renderer/releases>`__. Please send your logfiles to info@inexor.org.
-
-Please visit `inexor.org <https://inexor.org>`__ and join our `Discord <https://discord.gg/acUW8k7>`__ server.
-
-----
-
```

Comment:
As it is the main entry point into a project, you should have a small description what the project is about.

Also the license should be mentioned here.

Reply:
This will be resolved by including the FAQ and links under that section.

Reply:
Done.